appinfo.go

```diff
@@ -31,11 +34,13 @@ type AppengineInfo interface {
 }
 
 var (
-	zone           string
-	zoneMtx        sync.Mutex
-	config         *rest.Config
-	k8sClientSet   *kubernetes.Clientset
-	istioClientSet *istio.Clientset
+	zone                        string
+	zoneMtx                     sync.Mutex
+	config                      *rest.Config
+	k8sClientSet                *kubernetes.Clientset
+	istioClientSet              *istio.Clientset
+	cloudResourceManagerService *cloudresourcemanager.Service
+	syncMap                     sync.Map
```

Comment:
nit; i would probably rename this to something more like `projectToId` or similar